modules/datahandling/projectrequests.go

```diff
@@ -417,6 +414,23 @@ func (p projectLookupRequest) process(db dbfs.DBFS) ([]dhClosure, error) {
 	return []dhClosure{toSenderClosure{msg: res}}, nil
 }
 
+func projectLookup(senderID string, projectID int64, db dbfs.DBFS) (projectLookupResult, error) {
+
+	name, permissions, err := db.MySQLProjectLookup(projectID, senderID)
+
+	if err != nil {
+		return projectLookupResult{}, err
```

Comment:
this should be a nil return on error, not an empty object return. in this case, the object itself will exist but all of it's fields will get their default values, which isn't what we want because if the client sees it it won't make sense.